src/main/java/com/conveyal/datatools/manager/jobs/validation/SharedStopsValidator.java

```diff
@@ -88,26 +115,28 @@ public void validate() {
                     continue;
                 }
 
-                stopGroupStops.putIfAbsent(stopGroupId, new HashSet<>());
-                Set<String> seenStopIds = stopGroupStops.get(stopGroupId);
-
                 // Check for SS_01 (stop id appearing in multiple stop groups)
+                // Make sure this error is only returned if we are inside the feed that is being checked
                 if (seenStopIds.contains(stopId)) {
-                    registerError(stops
-                            .stream()
-                            .filter(stop -> stop.stop_id.equals(stopId))
-                            .findFirst()
-                            .orElse(stops.get(0)), NewGTFSErrorType.MULTIPLE_SHARED_STOPS_GROUPS
-                    );
+                    if (this.feedId.equals(sharedStopFeedId)) {
+                        registerError(stops
+                                .stream()
+                                .filter(stop -> stop.stop_id.equals(stopId))
+                                .findFirst()
+                                .orElse(new Stop()), NewGTFSErrorType.MULTIPLE_SHARED_STOPS_GROUPS
+                        );
+                    }
                 } else {
                     seenStopIds.add(stopId);
                 }
 
                 // Check for SS_02 (multiple primary stops per stop group)
-                if (stopGroupsWithPrimaryStops.contains(stopGroupId)) {
-                    registerError(NewGTFSError.forFeed(NewGTFSErrorType.SHARED_STOP_GROUP_MUTLIPLE_PRIMARY_STOPS, stopGroupId));
-                } else if (configReader.get(IS_PRIMARY_INDEX).equals("true")) {
-                    stopGroupsWithPrimaryStops.add(stopGroupId);
+                if (configReader.get(IS_PRIMARY_INDEX).equals("1") || configReader.get(IS_PRIMARY_INDEX).equals("true")) {
+                    if (stopGroupsWithPrimaryStops.contains(stopGroupId)) {
+                        registerError(NewGTFSError.forFeed(NewGTFSErrorType.SHARED_STOP_GROUP_MUTLIPLE_PRIMARY_STOPS, stopGroupId));
```

Comment:
This typo here or in the ErrorTypes too? **`Mutliple`**

Reply:
U r in Germany.. what would the germans do

Reply:
There are mutliple schools of thought

Reply:
resolved